.travis.yml

```diff
@@ -72,7 +73,7 @@ jobs:
     - stage: dependencies
       node_js: '10'
       if: type IN (cron, pull_request) AND NOT branch =~ ^greenkeeper/.*
-      env: TAV=generic-pool,mysql,mysql2,redis,koa-router,handlebars,mongodb-core
+      env: TAV=generic-pool,mysql,mysql2,redis,koa-router,handlebars,mongodb-core,cassandra-driver
```

Comment:
We should probably add this to the `ioredis,pg` build as that's currently the one that takes the shortest. This one seems to on average take the longest.

Reply:
Will do. 👍

Reply:
fixed.